Platon/backend/app/profile_management/views.py

```diff
@@ -107,6 +111,174 @@ def delete(user_id,self):
+                            500: 'Database Connection Problem'})
+        @api.expect(skills_post_parser)
+        def post(self):
+            """
+                Creates a skill
+            """
+            form = SkillsPostForm(request.form)
+            if form.validate():
+                new_skill = Skills(form.name.data)
+                try:
+                    db.session.add(new_skill)
+                    db.session.commit()
+                except:
+                    return make_response(jsonify({'error': 'Database Connection Problem'}), 500)
+                return make_response(jsonify({'msg': ' Successfully added'}), 201)
+            else:
+                return make_response(jsonify({'error': 'Wrong input format'}), 400)
+
+        @api.doc(responses={201: 'Successfully Updated', 400: 'Wrong Input Format', 401: 'Authentication Problem',
+                            500: 'Database Connection Problem'})
+        @api.expect(skills_put_parser)
+        def put(self):
+            """
+                Updates a skill
+            """
+            form = SkillsPutForm(request.form)
+            if form.validate():
+                try:
+                    skill = Skills.query.filter(Skills.id == form.id.data).first()
+                    if skill is None:
+                        return make_response(jsonify({'error': 'Please give an appropriate skill ID'}), 400)
+                    skill.name = form.name.data if form.name.data != '' else skill.name
+                    db.session.commit()
+                except:
+                    return make_response(jsonify({'error': 'Database Connection Problem'}), 500)
+                return make_response(jsonify({'msg': ' Successfully changed'}), 201)
+            else:
+                return make_response(jsonify({'error': 'Wrong input format'}), 400)
+
+        @api.doc(responses={200: 'Successfully Deleted', 400: 'Wrong Input Format', 401: 'Authentication Problem',
+                            500: 'Database Connection Problem'})
+        @api.expect(skills_delete_parser)
+        def delete(self):
+            """
+                Deletes a skill
+            """
+            form = SkillsDeleteForm(request.form)
+            if form.validate():
+                try:
+                    skill = Skills.query.filter(Skills.id == form.id.data).first()
+                    if skill is None:
+                        return make_response(jsonify({'error': 'Skill does not exist'}), 400)
+                    db.session.delete(skill)
+                    db.session.commit()
+                    return make_response(jsonify({'msg': 'Successfully Deleted'}), 200)
+                except:
+                    return make_response(jsonify({'error': 'Database Connection Problem'}), 500)
+            else:
+                return make_response(jsonify({'error': 'Wrong input format'}), 400)
+
```

Comment:
Same thing as Jobs part

Reply:
I added them in case of a user enters wrong input and she wants to change or delete them so that database holds no unnecessary data.